internal/app/cmd_serve.go

```diff
@@ -57,7 +57,6 @@ type serveCmd struct {
 
 type serveCommonConfig struct {
 	IngressBind         *url.URL             `help:"HTTP Ingress bind" default:"http://127.0.0.1:8891"`
-	Bind                *url.URL             `help:"Starting endpoint to bind to and advertise to for all FTL services" default:"http://127.0.0.1:8892"`
```

Comment:
I don't love this, as from a user perspective it's not obvious at all how to change the bind port. Why don't we just check that admin-endpoint == bind and error out?

Reply:
Thinking about it some more I think we should just completely ignore FTL_ENDPOINT in dev/serve mode. This will let users set FTL_ENDPOINT in hermit for deploy etc, but still run `ftl dev` normally.

Reply:
I have updated it do now FTL_ENDPOINT is ignored in dev/serve mode

Reply:
FYI the reason I want this it to make for a self contained `ftl test` command for JVM integration tests: https://github.com/block/ftl/pull/5427